dotCMS/src/main/java/com/dotmarketing/util/UtilMethods.java

```diff
@@ -3527,9 +3538,15 @@ public static boolean isAdminMode(HttpServletRequest request, HttpServletRespons
         return adminMode;
     }
 
-    public static JsonFieldTransformer getJsonFieldTransformer(
-            final List<com.dotmarketing.portlets.structure.model.Field> oldField) {
+    public static String getLayousAsJson(final Structure structure, final User user) {
+        try {
+            final StructureTransformer transformer = new StructureTransformer(structure);
+            final ContentType contentType = transformer.from();
 
-        return new JsonFieldTransformer(new LegacyFieldTransformer(oldField).asList());
+            final FieldLayout layout = APILocator.getContentTypeFieldLayoutAPI().getLayout(contentType.id(), user);
+            return MAPPER.writeValueAsString(layout.getRows());
+        } catch (DotDataException | DotSecurityException | JsonProcessingException e) {
+            throw new DotRuntimeException(e);
+        }
     }
```

Comment:
Please move this to the class `StructureWebApi` which is referenced in velocity as `$structures`.  It does not need to be static.

Reply:
ready